src/scene.js

```diff
@@ -936,6 +936,8 @@ Scene.prototype.removeTile = function (key) {
         tile.remove(this);
     }
 
+    this.workers.forEach(x => WorkerBroker.postMessage(x, 'removeTile', key));
```

Comment:
We don't want to call `removeTile` for every worker because the tile only resides in one worker. The tile knows its worker (`tile.worker`), and has a method for posting messages to its worker (https://github.com/tangrams/tangram/blob/iwillig/cancel-xhr-request/src/tile.js#L112). It also looks like `Tile.remove()` already calls `removeTile` on the correct thread (https://github.com/tangrams/tangram/blob/iwillig/cancel-xhr-request/src/tile.js#L286). Interestingly though, `Tile.remove()` doesn't appear to call `Tile.freeResources()`. So looks like we don't need a worker-specific call here, and the `Tile.remove()` call above should cover it, but another look at how `freeResources` fits in may be needed? (Sorry if I missed something here.)